### ISSUE TITLE
Cannot import library

### ISSUE THREAD

**Issue by deltakilomilo** [LayerZero-Labs/LayerZero#1]
Hi I'm trying to follow the instructions to import the library `"@layerzerolabs/contracts": "latest",` in my package.json, but it 
says it cannot be found. Here is the error trace when running npm install / yarn: 

```
yarn install v1.22.17
warning package.json: No license field
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
warning No license field
[1/4] 🔍  Resolving packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@layerzerolabs%2fcontracts: Not found".
info If you think this is a bug, please open a bug report with the information provided in "/<redacted>/app/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by fmhall** [comment]
Also running into this!

**Comment by mediavrog** [comment]
same here

**Comment by ihorbond** [comment]
such a package doesn't exist on npm

**Comment by XingyuLi20** [comment]
same error

**Comment by Akira-Taniguchi** [comment]
me too

**Comment by fmhall** [comment]
The packages are currently private - the team will publicly release them soon. In the meantime, you can copy the actual interfaces and libraries into your contracts folder.

**Comment by traitmeta** [comment]
Also running into this!

**Comment by swimmiee** [comment]
same here

**Comment by JeonghwanH** [comment]
Also running into this..!

**Comment by witwiki** [comment]
same here

**Comment by stavalfi** [comment]
2023.. is layer zero not maintained any more?

**Comment by calebbanister** [comment]
Yes it is still very much maintained but the contracts havent changed at all. This repo really just meant to show the contract code.

If you want to build on LayerZero , probably best to check out the solidity examples / tests here:

https://github.com/LayerZero-Labs/solidity-examples/tree/main/contracts

**Comment by tylerpashigian** [comment]
> Yes it is still very much maintained but the contracts havent changed at all. This repo really just meant to show the contract code.
> 
> If you want to build on LayerZero , probably best to check out the solidity examples / tests here:
> 
> https://github.com/LayerZero-Labs/solidity-examples/tree/main/contracts

In the main docs there are instructions to be able to download the LZ contracts by adding `"@layerzerolabs/contracts": "latest",` to `package.json` and installing the package. This resulted in a 404 for me. Is this package private, or am I doing something wrong? I downloaded the contracts manually, but would like to follow "best practice". Thanks!
